### PR TITLE
[UBS] - When we make an order on ubs, the corresponding courier and his locations have to set up #5415

### PR DESCRIPTION
**Pull request by ABbondar**
# GreenCityUBS PR
https://github.com/ita-social-projects/GreenCity/issues/5415

## Summary Of Changes :fire:
Changed method for getting all active locations where courier is working in Order controller
Changed method for getting info about tariff in Order controller

## Issue Link :clipboard:
_&lt;Link to the issue&gt;_

## Added
* _&lt;detail item of what was added&gt;_
* _&lt;describe functionality that was added&gt;_

## Changed
* _&lt;detail item of what was changed&gt;_
* _&lt;describe functionality that was changed&gt;_

## Deleted
* _&lt;detail item of what was removed&gt;_

## How to test :clipboard:
It can be tests via swagger in order-controler, endpoints - ubs/locations/{courierId} and ubs/tariffinfo/{locationId}

# PR Checklist Forms

_(to be filled out by PR submitter)_
- [ ] Code is up-to-date with the `dev` branch.
- [x] You've successfully built and run the tests locally.
- [x] There are new or updated unit tests validating the change.
- [x] JIRA/ Github Issue number & title in PR title (ISSUE-XXXX: Ticket title)
- [x] This template filled (above this section).
- [x] Sonar's report does not contain bugs, vulnerabilities, security issues, code smells ar duplication
- [x] `NEED_REVIEW` and `READY_FOR_REVIEW` labels are added.
- [x] All files reviewed before sending to reviewers

### CODE REVIEW

**Review by LiliaMokhnatska — APPROVED** [review]

**Review by juseti — APPROVED** [review]

**Review by MaksymGolik — APPROVED** [review]

File: core/src/main/java/greencity/controller/OrderController.java [hunk 1]
```diff
@@ -513,47 +513,55 @@ public ResponseEntity<HttpStatus> receivePaymentClient(
...
      * @param changeLoc - optional param. If it is present controller will return
      *                  info about locations
+     * @param courierId - id of courier
      * @param uuid      - user's uuid
      * @return {@link OrderCourierPopUpDto}
+     * @author Anton Bondar
      */
-    @ApiOperation(value = "Get all location where courier is working")
+    @ApiOperation(value = "Get all active locations where courier is working")
+    @GetMapping("/allLocations/{courierId}")
```
Comment (VadymMakitra) [inline core/src/main/java/greencity/controller/OrderController.java]:
```suggestion
    @GetMapping("/locations/{courierId}")
```

File: core/src/main/java/greencity/controller/OrderController.java [hunk 2]
```diff
@@ -513,47 +513,55 @@ public ResponseEntity<HttpStatus> receivePaymentClient(
...
     @ApiResponses(value = {
         @ApiResponse(code = 200, message = HttpStatuses.OK),
         @ApiResponse(code = 400, message = HttpStatuses.BAD_REQUEST),
         @ApiResponse(code = 401, message = HttpStatuses.UNAUTHORIZED),
         @ApiResponse(code = 404, message = HttpStatuses.NOT_FOUND)
     })
-    @GetMapping("/tariffinfo-for-location/{locationId}")
-    public ResponseEntity<OrderCourierPopUpDto> getInfoAboutTariff(@Valid @PathVariable Long locationId) {
-        return ResponseEntity.status(HttpStatus.OK).body(ubsClientService.getTariffInfoForLocation(locationId));
+    @GetMapping("/tariffinfo-for-location")
```
Comment (VadymMakitra) [inline core/src/main/java/greencity/controller/OrderController.java]:
```suggestion
    @GetMapping("/tariffinfo/location/{locationId}")
```

Comment (ABbondar) [reply core/src/main/java/greencity/controller/OrderController.java]: edited

File: core/src/main/java/greencity/controller/OrderController.java (hunk 1 above)
Comment (ABbondar) [reply core/src/main/java/greencity/controller/OrderController.java]: edited

**Pull request merged**